.travis.yml

```diff
@@ -108,9 +108,9 @@ script:
     - /tmp/julia/bin/julia --precompiled=no -e 'true' &&
         /tmp/julia/bin/julia-debug --precompiled=no -e 'true'
     - /tmp/julia/bin/julia -e 'versioninfo()'
-    - export JULIA_CPU_CORES=2 && export JULIA_TEST_MAXRSS_MB=600 && cd /tmp/julia/share/julia/test &&
```

Comment:
it's an extra level of nested spawning on Julia calls, and having to worry about whether the command-line flags are all getting propagated as we want them